api/v1/types.proto

```diff
@@ -43,6 +43,7 @@ message Volume {
     string Usage = 13;
     // inline volumes are not support anymore. need to remove field in the next version
     bool Ephemeral = 14;
+    bool Mounted = 15;
```

Comment:
Can we use `MountStatus` instead of `Mounted`? BTW should it be as part of `Spec` or `Status`?